test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -216,7 +216,7 @@ BOOST_AUTO_TEST_CASE(smoke_test)
 	char const* text = R"(
 		contract test {
 			uint256 stateVariable1;
-			function fun(uint256 arg1) { uint256 y; }
+			function fun(uint256) { uint256 y; }
```

Comment:
This should still flag `y` as unused. Can you change that to use the variables instead?